build.xml

```diff
@@ -55,8 +55,8 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
     <property name="jackson-mapper-asl.version" value="1.9.11"/>
     <property name="dependency-check-ant.version" value="2.1.0"/>
 
-    <property name="commons-io.version" value="2.4"/>
-    <property name="kerby.version" value="1.0.0-RC2"/>
+    <property name="commons-io.version" value="2.5"/>
+    <property name="kerby.version" value="1.0.0"/>
```

Comment:
1.1.0 is available - why not update to that?

Reply:
Because it is not available for all submodules of Kerby, it would make the patch more complex.

Reply:
I will try again to update to 1.1.0

Reply:
Changing to 1.1.0 worked for me - it compiled and the tests ran. Is there something else?

I did try 1.1.1 but that had the issue you mentioned.